qa/tasks/parent_cache.py

```diff
@@ -0,0 +1,563 @@
+"""
+parent_cache task
```

Comment:
Nit: can we rename this task to 'immutable_object_cache' instead?

Reply:
done